README.md

```diff
@@ -23,7 +23,7 @@ After you're finished please remove all the comments and instructions!
 -->
 
 <div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
```

Comment:
- [ ] Kindly remove the Microverse logo. If you have your logo, you can use it here. Otherwise, you should remove it. 🙏